packages/edit-widgets/src/components/layout/index.js

```diff
@@ -21,25 +22,27 @@ function Layout( { blockEditorSettings } ) {
 	const [ selectedArea, setSelectedArea ] = useState( null );
 	return (
 		<SlotFillProvider>
```

Comment:
This screen could reuse the EditorRegions component (if we find a way to expose it properly) and avoid duplicate a lot of CSS.